demisto_sdk/commands/common/tests/tools_test.py

```diff
@@ -122,6 +137,32 @@ def test_find_type_ignore_sub_categories(self):
         assert output == FileType.INTEGRATION, \
             f'find_type({VALID_BETA_INTEGRATION_PATH}) returns: {output} instead {FileType.INTEGRATION}'
 
+    def test_find_type_with_invalid_yml(self, malformed_integration_yml):
+        """
+        Given
+        - A malformed yml file.
+
+        When
+        - Running find_type.
+
+        Then
+        - Ensure no exception/error is raised and None is returned.
+        """
+        assert not find_type(malformed_integration_yml.yml.path, ignore_invalid_schema_file=True)
+
```

Comment:
Lets try&except and fail the test gracefully and have a good failure message.

Reply:
will do